browser-features/chrome/common/command-palette/style.css

```
@@ -0,0 +1,110 @@
+
+.command-palette-item {
+  display: flex;
+  align-items: center;
+  padding: 8px 16px;
+  cursor: pointer;
+  gap: 12px;
+  user-select: none;
+}
+
+.command-palette-item:hover {
+  background: color-mix(in srgb, currentColor 8%, transparent);
```

Stylelint (17.9.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)